Dockerfile

```diff
@@ -16,18 +16,26 @@
 FROM golang:1.12.5 as builder
 
 ARG ARCH
+ARG USERNAME=cluster-api-manager
+ARG GROUPNAME=cluster-api-manager
 
 # Copy in the go src
-WORKDIR $GOPATH/src/sigs.k8s.io/cluster-api
+WORKDIR ${GOPATH}/src/sigs.k8s.io/cluster-api
 COPY pkg/    pkg/
 COPY cmd/    cmd/
 COPY vendor/ vendor/
 
+#create non-root user
+RUN groupadd -g 999 ${GROUPNAME} && \
+    useradd -r -u 999 -g ${GROUPNAME} ${USERNAME}
```

Comment:
Since the UID/GID really isn't significant on the host here, we can avoid the group/user manipulation and just use the pre-existing `nobody` user.

```
> docker run --entrypoint=sh -ti --rm gcr.io/distroless/base:debug
/ # cat /etc/passwd 
root:x:0:0:root:/home:/sbin/nologin
nobody:x:65534:65534:nobody:/nonexistent:/sbin/nologin
```

Reply:
Sure. I didn't know that `nobody` was pre-created. Thanks